From andre@bitwigglers.org  Mon Nov 26 22:59:56 2007
From: Andre Haupt <andre@bitwigglers.org>
Date: Tue, 20 Nov 2007 11:41:37 +0100
Subject: USB: cyberjack: remove a unnecessary variable.
To: linux-usb-devel@lists.sourceforge.net
Cc: greg@kroah.com, support@reiner-sct.com, Andre Haupt <andre@bitwigglers.org>
Message-ID: <11955552973227-git-send-email-andre@bitwigglers.org>

From: Andre Haupt <andre@bitwigglers.org>

This also fixes a sparse warning that symbol 'result' shadows an earlier one.

Signed-off-by: Andre Haupt <andre@bitwigglers.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/cyberjack.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/serial/cyberjack.c
+++ b/drivers/usb/serial/cyberjack.c
@@ -319,7 +319,6 @@ static void cyberjack_read_int_callback(
 	/* React only to interrupts signaling a bulk_in transfer */
 	if( (urb->actual_length==4) && (data[0]==0x01) ) {
 		short old_rdtodo;
-		int result;
 
 		/* This is a announcement of coming bulk_ins. */
 		unsigned short size = ((unsigned short)data[3]<<8)+data[2]+3;
